ddtrace/opentracer/tags.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+
+TAG_NAMES = [
+    # OpenTracing tags
```

Comment:
Do we need to copy these tags from OT? can't we import OTTags like you did before and leave only Datadog tags in our module?